functions.opengraph.php

```diff
@@ -11,6 +11,10 @@
 add_action( 'wp_head', 'jetpack_og_tags' );
 
 function jetpack_og_tags() {
+	echo jetpack_og_get_tags();
+}
+
+function jetpack_og_get_tags() {
```

Comment:
I'm not sure I understand your reasoning behind creating this new function. Could you explain? Was this to make unit testing easier?